We predict the consequence of variants (germline or somatic) on genes associated with diseases based on the evidence for the Genetic associations and Somatic mutation data types. This effect is defined by consequence terms from the [Sequence Ontology](http://www.sequenceontology.org/) (SO) project.

The SO consequence terms, their descriptions and accession IDs are shown in the table below:

<table class="table">

<thead>

<tr>

<th>SO term</th>

<th>Definitions</th>

<th>SOÂ accession</th>

</tr>

</thead>

<tbody>

<tr>

<td>3 prime UTR variant</td>

<td>A UTR variant of the 3' UTR.</td>

<td>[SO_0001624](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001624)</td>

</tr>

<tr>

<td>5 prime UTR variant</td>

<td>A UTR variant of the 5' UTR.</td>

<td>[SO_0001623](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001623)</td>

</tr>

<tr>

<td>coding sequence variant</td>

<td>A sequence variant that changes the coding sequence.</td>

<td>[SO_0001580](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001580)</td>

</tr>

<tr>

<td>frameshift variant</td>

<td>A sequence variant which causes a disruption of the translational reading frame, because the number of nucleotides inserted or deleted is not a multiple of three.</td>

<td>[SO_0001589](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001589)</td>

</tr>

<tr>

<td>inframe deletion</td>

<td>An inframe non synonymous variant that deletes bases from the coding sequence.</td>

<td>[SO_0001822](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001822)</td>

</tr>

<tr>

<td>inframe insertion</td>

<td>An inframe non synonymous variant that inserts bases into in the coding sequence.</td>

<td>[SO_0001821](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001821)</td>

</tr>

<tr>

<td>initiator codon variant</td>

<td>A codon variant that changes at least one base of the first codon of a transcript.</td>

<td>[SO_0001582](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001582)</td>

</tr>

<tr>

<td>intron variant</td>

<td>A transcript variant occurring within an intron.</td>

<td>[SO_0001627](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001627)</td>

</tr>

<tr>

<td>missense variant</td>

<td>A sequence variant that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.</td>

<td>[SO_0001583](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001583)</td>

</tr>

<tr>

<td>nearest gene counting from 5' end</td>

<td>Nearest gene to variant counting from 5' end.</td>

</tr>

<tr>

<td>non coding transcript exon variant</td>

<td>A sequence variant that changes non-coding exon sequence in a non-coding transcript.</td>

<td>[SO_0001619](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001619)</td>

</tr>

<tr>

<td>regulatory region variant</td>

<td>A sequence variant located within a regulatory region.</td>

<td>[SO_0001566](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001566)</td>

</tr>

<tr>

<td>splice acceptor variant</td>

<td>A splice variant that changes the 2 base region at the 3' end of an intron.</td>

<td>[SO_0001574](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001574)</td>

</tr>

<tr>

<td>splice donor variant</td>

<td>A splice variant that changes the 2 base pair region at the 5' end of an intron.</td>

<td>[SO_0001575](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001575)</td>

</tr>

<tr>

<td>splice region variant</td>

<td>A sequence variant in which a change has occurred within the region of the splice site, either within 1-3 bases of the exon or 3-8 bases of the intron.</td>

<td>[SO_0001630](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001630)</td>

</tr>

<tr>

<td>stop gained</td>

<td>A sequence variant whereby at least one base of a codon is changed, resulting in a premature stop codon, leading to a shortened transcript.</td>

<td>[SO_0001587](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001587)</td>

</tr>

<tr>

<td>stop lost</td>

<td>A sequence variant where at least one base of the terminator codon (stop) is changed, resulting in an elongated transcript.</td>

<td>[SO_0001578](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001578)</td>

</tr>

<tr>

<td>stop retained variant</td>

<td>A sequence variant where at least one base in the terminator codon is changed, but the terminator remains.</td>

<td>[SO_0001567](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001567)</td>

</tr>

<tr>

<td>synonymous variant</td>

<td>A sequence variant where there is no resulting change to the encoded amino acid.</td>

<td>[SO_0001819](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001819)</td>

</tr>

<tr>

<td>upstream gene variant</td>

<td>A sequence variant located 5' of a gene.</td>

<td>[SO_0001631](http://www.ontobee.org/ontology/SO?iri=http://purl.obolibrary.org/obo/SO_0001631)</td>

</tr>

</tbody>

</table>